app/src/main/java/org/oppia/app/utility/FontScaleConfigurationUtil.kt

```diff
@@ -0,0 +1,33 @@
+package org.oppia.app.utility
+
+import android.content.Context
+import android.content.Context.WINDOW_SERVICE
+import android.util.DisplayMetrics
+import android.view.WindowManager
+import org.oppia.app.model.StoryTextSize
+
+object FontScaleConfigurationUtil {
+  fun adjustFontSize(
```

Comment:
Please add a KDoc explaining how to use this method & what it does.

Reply:
Ditto